source/ecommerce-cards-1.30-nl.yaml

```diff
@@ -473,7 +473,7 @@ paragraphs:
     text: "De inhoud is voornamelijk afkomstig uit het SCP."
   -
     value: "T00500"
-    text: "Mappings"
```

Comment:
In Dutch we'd use "mapping" in Dutch too, to talk about the kind of mapping we mean here.